js/script.js

```diff
@@ -24,6 +24,12 @@ $Router.config([
                 })
             });
         }
+
+        let anchorList = Array.from(document.querySelectorAll('a[r-href]'));
+        anchorList.forEach(anchor => {
+            let url = anchor.getAttribute('r-href');
+            anchor.setAttribute('href', '#' + url);
+        });

```

Comment:
Without this forEach link's on pages doesn't work. 
You can check it, for example, add link `<a r-href="home">Installation</a>` at `partial/options.html`.